features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt

```diff
@@ -284,34 +289,61 @@ private fun LazyListScope.roomMemberListSection(
     items(members.orEmpty()) { matrixUser ->
         RoomMemberListItem(
             modifier = Modifier.fillMaxWidth(),
-            roomMember = matrixUser,
-            onClick = { onMemberSelected(matrixUser) }
+            roomMemberWithIdentity = matrixUser,
+            onClick = { onMemberSelected(matrixUser.roomMember) }
         )
     }
 }
 
 @Composable
 private fun RoomMemberListItem(
-    roomMember: RoomMember,
+    roomMemberWithIdentity: RoomMemberWithIdentityState,
     onClick: () -> Unit,
     modifier: Modifier = Modifier,
 ) {
-    val roleText = when (roomMember.role) {
+    val roleText = when (roomMemberWithIdentity.roomMember.role) {
         RoomMember.Role.ADMIN -> stringResource(R.string.screen_room_member_list_role_administrator)
         RoomMember.Role.MODERATOR -> stringResource(R.string.screen_room_member_list_role_moderator)
         RoomMember.Role.USER -> null
     }
+
     MatrixUserRow(
         modifier = modifier.clickable(onClick = onClick),
-        matrixUser = roomMember.toMatrixUser(),
+        matrixUser = roomMemberWithIdentity.roomMember.toMatrixUser(),
         avatarSize = AvatarSize.UserListItem,
-        trailingContent = roleText?.let {
-            @Composable {
-                Text(
-                    text = it,
-                    style = ElementTheme.typography.fontBodySmRegular,
-                    color = ElementTheme.colors.textSecondary,
-                )
+        trailingContent = {
```

Comment:
This is not new, but I think we will need to add some padding at the start of the trailing content:

<img width="337" alt="image" src="https://github.com/user-attachments/assets/c681c241-6f6b-41b2-a724-148932523403" />

Reply:
I think some padding is added by default:

<img width="396" alt="image" src="https://github.com/user-attachments/assets/2ac0eb1c-859e-40a3-9eb0-e688de1655e9" />